### PR TITLE
Spectrum

### PR DESCRIPTION
**Pull request by Rob-Manders**
### New Stylesheet Submission:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [x] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

This is my first Style Stage submission and my first ever GitHub pull request. I think I've done it right, but please let me know if I haven't so I can correct it. Thank you. :)

### CODE REVIEW

File: src/_data/styles/spectrum.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"title": "Spectrum",
+	"author": "Robert Manders",
+	"stylesheet": "https://github.com/Rob-Manders/Style-Stage-Spectrum/blob/main/spectrum.css",
```
Comment (5t3ph) [inline src/_data/styles/spectrum.json:4]: Little update to use the "raw" file. 👍
```suggestion
	"stylesheet": "https://raw.githubusercontent.com/Rob-Manders/Style-Stage-Spectrum/main/spectrum.css",
```

**Review by 5t3ph — CHANGES_REQUESTED** [review]

**Comment by 5t3ph** [comment]
@Rob-Manders really lovely!

Small issue - the purple used on `:hover` and `:focus` for links is juuuust barely missing contrast, it needs to be a few clicks lighter to pass at 4.5:1 for the size or your text links.

Let me know when that's updated and I'll redeploy the preview, found here:
https://deploy-preview-92--moderncss-styles.netlify.app/styles/spectrum/

**Comment by Rob-Manders** [comment]
@5t3ph thank you for letting me know. I've lightened it up so it should be reading at 4.95 now.

**Review by 5t3ph — APPROVED** [review]
@Rob-Manders perfect, publishing now!

Thank you for participating! If you haven't already, consider subscribing for updates to learn about new styles and features: https://stylestage.dev/subscribe/

**Pull request merged**